### PR TITLE
rg-33: Images CRUD to development

### PR DESCRIPTION
**Pull request by vlad-blip**

### CODE REVIEW

**Review by nickolay-grechkin — APPROVED** [review]

File: backend/src/bundles/profile/profile.repository.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ProfileRepository implements Partial<IRepository> {
         lastName: string;
         transaction: Transaction;
     }): Promise<Profile> {
-        return this.profileModel
+        return await this.profileModel
```
Comment (Vitaliy-Kalenichenko) [inline backend/src/bundles/profile/profile.repository.ts:24]: Why is the `await` added here?

**Review by Vitaliy-Kalenichenko — CHANGES_REQUESTED** [review]
It's backend related task, so there should not be any changes in the frontend folder

**Comment by Vitaliy-Kalenichenko** [comment]
Consider limiting the file size. Perhaps 10 MB will be enough.

File: backend/src/bundles/profile/profile.repository.ts (hunk 1 above)
Comment (vlad-blip) [reply backend/src/bundles/profile/profile.repository.ts:24]: because typescript told me so

**Comment by vlad-blip** [comment]
> It's backend related task, so there should not be any changes in the frontend folder

I haven't made any significant changes to frontend folder. I just ran `npm run prettify` command to fix all styling issues, because lint wouldn't let me commit anything while having file styling errors, and deleted the export duplicates. I didn't touch any logic on frontend.

**Comment by Vitaliy-Kalenichenko** [comment]
Please revert the commit in which you ran `npm run prettify`. And don't do it again. If you need to run linters ask about a task for it

**Pull request closed without merging**